src/odin/generate/cubic/ods_fact.py

```diff
@@ -369,7 +369,7 @@ def load_cdc_records(self) -> int:
                     start_odin_index, start_odin_index + insert_df.height, dtype=pl.Int64()
                 ).alias("odin_index"),
                 pl.lit(self.history_snapshot, dtype=pl.String()).alias("odin_snapshot"),
-            ).drop(self.history_drop_columns)
```

Comment:
these extra .drop params are to fix some errors that popped up in DEV after the last PR

Reply:
Can you add to description and/or PR name please? That way we can trace the change